### PR TITLE
Upload middleware file by front-end.

### PR DESCRIPTION
**Pull request by dijalmasilva**
**Describe feature**
Upload middleware file through the front-end, list all middlewares persisted and limit size file in 25MB.

### CODE REVIEW

File: heimdall-frontend/src/containers/Middlewares.js [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+import {connect} from 'react-redux'
+import {Upload, message, Icon, Input, Row, Card, Table, Pagination, Tag} from 'antd'
+import {save, initLoading, getMiddlewares} from '../actions/middlewares'
+
+const Dragger = Upload.Dragger
+const Column = Table.Column
+
+class Middlewares extends Component {
+
+    state = {
```
Comment (MAAARKIN) [inline heimdall-frontend/src/containers/Middlewares.js]: @dijalmasilva  you add state's that aren't used. plz, remove they

File: heimdall-frontend/src/containers/Middlewares.js [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+            </div>
+        )
+    }
+}
+
+
+const mapStateToProps = state => {
+    return {
+        middlewares: state.middlewares.middlewares,
+        interceptors: state.interceptors.interceptors,
```
Comment (MAAARKIN) [inline heimdall-frontend/src/containers/Middlewares.js]: @dijalmasilva plz remove states that aren't used too. plz

File: heimdall-frontend/src/containers/Middlewares.js (hunk 1 above)
Comment (dijalmasilva) [reply heimdall-frontend/src/containers/Middlewares.js]: Fixed.

Comment (dijalmasilva) [reply heimdall-frontend/src/containers/Middlewares.js]: Fixed

**Pull request merged**